qa/tasks/cephfs/test_subvolume.py

```diff
@@ -0,0 +1,132 @@
+import logging
+
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+from teuthology.exceptions import CommandFailedError
+
+log = logging.getLogger(__name__)
+
+
+class TestSubvolume(CephFSTestCase):
+    def setUp(self):
+        super(CephFSTestCase, self).setUp()
+
+    def tearDown(self):
+        super(CephFSTestCase, self).tearDown()
+
+    def test_subvolume(self):
```

Comment:
We definitely need to break this test down.  This unit test tests out so many different aspects. Please create a tracker ticket for code clean up.

Maybe add a doc string for this test. "test_subvolume" doesn't say much.
"""
tests the effects of setting "ceph.dir.subvolume" xattr on a FS subvolume.
"""

Reply:
test has now been broken down into 9 individual tests with relevant doc strings

Reply:
Thanks for doing this. I was only expecting a tracker ticket for later clean up. :-)